ckanext/protected_resources/plugin.py

```diff
@@ -186,6 +183,7 @@ def sysadmin_email(self):
     # ITemplateHelpers
     def get_helpers(self):
         return {
+            'resource_has_protected_status': resource_has_protected_status,
```

Comment:
1/10
Some helpers/auth methods feel like they could potentially be in a seperate file and referenced here instead?

Reply:
They could be, and there is a pattern of doing that in plugins that implement more helpers/auth/logic functions.